Library/Formula/spiff.rb

```diff
@@ -0,0 +1,15 @@
+class Spiff < Formula
+  desc "Tool and declarative YAML templating system specially for BOSH manifests."
+  homepage "https://github.com/cloudfoundry-incubator/spiff"
+  url "https://github.com/cloudfoundry-incubator/spiff/releases/download/v1.0.7/spiff_darwin_amd64.zip"
```

Comment:
We have a `godep` formula so it wouldn't actually be problematic to build using their preferred method of vendoring rather than fetching.

However, if they'd literally tell us to go away every time we ever sent someone upstream with a bug report because we prefer to build from source and supply our own binaries this may be a better fit for https://github.com/caskroom/homebrew-cask.